### PR TITLE
Development webpack fork

### PR DESCRIPTION
**Pull request by DominikGuzei**
This PR tracks the development of our new webpack setup which changes most of the project setup and thus creates a " hard fork" that cannot be merged / compared directly. It is up-to-date with our current `master` but we need to port over all open PRs against `master` previously & anything new we develop.

For short term releases we should still use the normal `development` / `master` branches until we feel confident to switch over (when this new branch is well tested and we are happy with the refactor). If we know that a feature takes longer to develop we should probably switch over to build it with the new structure in this branch!

**PRs to port over (IN THIS ORDER !!!):**
- [x] [Chore/ddw 79 Update Electron to version 1.7.11 …](https://github.com/input-output-hk/daedalus/pull/677) Ported PR: #684 (**merged**)
- [x] [Chore/ddw 80 Update for the about screen](https://github.com/input-output-hk/daedalus/pull/680) Ported PR: #686 (**merged**)
- [x] [Chore/ddw 82 Ada redemption tests in mainnnet mode](https://github.com/input-output-hk/daedalus/pull/681) Ported PR: #687 (**merged**)
- [x] [Chore/ddw 83 Move log to pub folder …](https://github.com/input-output-hk/daedalus/pull/682) Ported PR: #683 (**merged**)
- [x] [Fix/ddw 72 Update copy for the sync error screen](https://github.com/input-output-hk/daedalus/pull/660) Ported PR: #688 (**merged**)
- [x] [Fix/ddw 76 Time of your machine is different  …](https://github.com/input-output-hk/daedalus/pull/678) Ported PR #705 (**merged**)
- [x] [Fix/ddw 61 Detect when wallet is out of sync](https://github.com/input-output-hk/daedalus/pull/689)  Ported PR #706 (**merged**)
- [x] [Fix/ddw 84 Improve connecting reconnecting messages …](https://github.com/input-output-hk/daedalus/pull/696) Ported PR #707 (**merged**)
- [x] [Chore/ddw 101 Run acceptance tests on release/0.9.0 …](https://github.com/input-output-hk/daedalus/pull/721)  Ported PR: #721 (**merged**)
- [x] [Fix/DDW-93 Poll local time difference every 1 h …](https://github.com/input-output-hk/daedalus/pull/719) Ported PR: #754 (**merged**)
- [x] [Fix/ddw 119 Fix wallet restore reactions configuration](https://github.com/input-output-hk/daedalus/pull/735) Ported PR: #755 (**merged**)
- [x] [Feature/ddw 5 Send bug reports with logs from Daedalus](https://github.com/input-output-hk/daedalus/pull/585) Ported PR: #753 (**merged**)
- [x] [Fix/ddw-124 dont show validation error after …](https://github.com/input-output-hk/daedalus/pull/738) Ported PR: #756 (**merged**)
- [x] [Fix/ddw 125 fix broken get logs function on windows](https://github.com/input-output-hk/daedalus/pull/742) Ported PR: #757 (**merged**)
- [x] [Fix/ddw 139 Show time sync error even if …](https://github.com/input-output-hk/daedalus/pull/752) Ported PR: #758 (**merged**)
- [x] [[DEVOPS-676] bake the REPORT_URL into the binary …](https://github.com/input-output-hk/daedalus/pull/724) Ported PR: #759 (**merged**)
- [x] [Release/0.9.1 ](https://github.com/input-output-hk/daedalus/pull/784) - Ported PR: #792 (**merged**)
- [x] [Fix/ddw 168 Upgrade moment.js to latest version](https://github.com/input-output-hk/daedalus/pull/782) - Ported PR: #793 (**merged**)
- [x] [[DEVOPS-727] Mitigation](https://github.com/input-output-hk/daedalus/pull/800) - Ported PR: #802

**WIP PRs to be ported to new fork**
- [ ] [Feature/ddw 68 Account features implementation](https://github.com/input-output-hk/daedalus/pull/616) [**WIP**] - should not be merged!
- [ ] [Fix/ddw 70 Duplicate transaction ids for ETC](https://github.com/input-output-hk/daedalus/pull/699) [**WIP**] - should not be merged!
- [ ] [Feature/daef 416 Updates for the send money screen](https://github.com/input-output-hk/daedalus/pull/435) [**WIP**] - should not be merged!
- [ ] [Feature wire up chart to staking page](https://github.com/input-output-hk/daedalus/pull/64) [**WIP**] - should not be merged!

**Tasks:**
- [x] Fix theming for react-polymorph "Options" component (fixed in #709)
- [x] Fix issues with translation manager and generation of translation messages - Fixed in #692
- [x] Test if the updated installers etc. work with the fork!
- [x] After all PRs the latest `develop` branch including the release has to be merged into this branch and all remaining conflicts resolved.
- [x] Fix acceptance-tests issues

### CODE REVIEW

**Comment by DominikGuzei** [comment]
@cleverca22 @rvl the CI builds are failing again!

**Comment by rvl** [comment]
We merged #789 (DEVOPS-673) and now there are a few merge conflicts.
I have resolved the merge on the [rvl/development-webpack-fork-merge](https://github.com/input-output-hk/daedalus/commits/rvl/development-webpack-fork-merge) branch.
We can apply that commit to this branch when/if it passes CI.

**Comment by nikolaglumac** [comment]
Thanks for the info @rvl!
I'm afraid that your branch hasn't passed CI :(
What should we do?

**Comment by rvl** [comment]
Both @cleverca22 and I have independently merged and found an incompatibility with the updated nixpkgs revision and new dependencies of `make-installer`. I have pushed a fix for that to my branch.

**Comment by cleverca22** [comment]
ive merged most of @rvl 's changes into this branch, but its not obvious why it fails to run nsis on windows

File: source/main/utils/setupLogging.js [hunk 1]
```diff
@@ -9,12 +9,12 @@ export const setupLogging = () => {
   const logFilePath = path.join(pubLogsFolderPath, APP_NAME + '.log');
   ensureDirectoryExists(pubLogsFolderPath);
 
-  Log.transports.console.level = 'warn';
-  Log.transports.rendererConsole.level = 'warn';
-  Log.transports.file.level = 'debug';
-  Log.transports.file.maxSize = 20 * 1024 * 1024;
-  Log.transports.file.file = logFilePath;
-  Log.transports.file.format = '[{y}-{m}-{d} {h}:{i}:{s}.{ms} {z}] [{level}] {text}';
+  log.transports.console.level = false;
```
Comment (nikolaglumac) [inline source/main/utils/setupLogging.js]: @darko-mijic @DominikGuzei `console` transport has to be disabled as it causes `macOS` installer issues (Alert dialog is shown on first `log.error` call)

**Comment by DominikGuzei** [comment]
Thanks @nikolaglumac and congrats everyone! 🎉

**Pull request merged**